pkg/provider/aws/infra_template.go

```diff
@@ -1,9 +1,44 @@
 package aws
 
 import (
+	"fmt"
+
 	"github.com/klothoplatform/klotho/pkg/core"
+	"go.uber.org/zap"
 )
 
 func (a *AWS) Translate(result *core.ConstructGraph, dag *core.ResourceGraph) (Links []core.CloudResourceLink, err error) {
+	fmt.Println(dag.ListConstructs())
+	fmt.Println(dag.ListDependencies())
+
+	log := zap.S()
+
+	rootConstructs := result.GetRoots()
+	for _, construct := range rootConstructs {
```

Comment:
How come only the roots?

Reply:
Right now the graph is purely just a single lambda and generating from there. I need to change this so it will support more, but focused more on just getting the generation done for lambda so we could try to wire it all together. Will make the changes to add more after this pr